public/stylesheets/style.css

```diff
@@ -110,3 +110,7 @@ body{
 .form__help-block--hidden {
   display: none;
 }
+
+.starred {
```

Comment:
The style on workflowy is different - please change to match styles.  As well, this rule should use BEM notation (in this case, starred is a modified on some block and element).